### PR TITLE
answered questions for Sprint Challenge.

### PR DESCRIPTION
**Pull request by fncischen**
@ryanboris - initial commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,14 +34,36 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+*Adapative websites provide different fixed static layouts of a website based on different fixed parameters (i.e. min / max screen width).
```
Comment (secantsquared) [inline README.md:37]: This is a nice overview of each type of site.

File: README.md [hunk 2]
```diff
@@ -34,14 +34,36 @@ Edit this document to include your answers after each question. Make sure to lea
...
 
+*Setting the font size to 62.5% means setting the base browser font size from its default value to 62.5% of its default value. 
+For rem units, it means that the base rem unit (1 rem) has been reduced to 62.5% of its original value.
+
+Speaking concretely, if the base browser font size is 16 pixels, 62.5% of 16 is 10 pixels. 1 rem has changed from 16 pixels to 10 pixels.
+Which means, any child element/class/id will have a different base font size, and the rem will be adjusted accordingly.*
+
 4. How would you describe preprocessing to someone new to CSS?
 
+* Preprocessing is another user-friendly way of organizing your Style Sheets for your HTML browser / elements.
```
Comment (secantsquared) [inline README.md]: Nicely said.

File: README.md [hunk 3]
```diff
@@ -34,14 +34,36 @@ Edit this document to include your answers after each question. Make sure to lea
...
+
+I think of it as a notepad to sketch your ideas in a clear, organized fashion, which then is sent to a compiler - which acts like a translator to translate your 
+ideas using the Less.JS language - to return a official CSS.file that will render each specific HTML browsesr element/class/id correctly. *  
+
 5. What is your favorite concept in preprocessing? What is the concept that gives you the most trouble?
 
+* I like how Preprocessing is a very modular means of designing user interfaces - it's very clean, and easy to debug / isolate problems. 
+The syntax does take some time to get used to, compared to regular CSS!*
+
+
```
Comment (secantsquared) [inline README.md:65]: Great answers - to the point and phrased well.

File: less/index.less [hunk 4]
```diff
@@ -1,7 +1,12 @@
 // Import your less files here:
 
-
-
+@import: "variables.less";
```
Comment (secantsquared) [inline less/index.less]: good structure for `@imports` and organized properly

File: css/index.css [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+  font-size: 4rem;
+}
+h2 {
+  font-size: 3.2rem;
+}
+h3 {
+  font-size: 2.8rem;
+}
+p {
+  font-size: 1.6rem;
```
Comment (secantsquared) [inline css/index.css:170]: Great use of responsive units here.

File: css/index.css [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+  padding: 20px 0;
+  border: 1px double #e0e0e0;
+  border-radius: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.menu {
+  display: flex;
+  flex-wrap: wrap;
```
Comment (secantsquared) [inline css/index.css:223]: in the future you can combine these two using `flex-flow`
```suggestion
.menu {
    flex-flow:  column wrap
}
```

File: css/index.css [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+}
+.home .walk .walk-img {
+  width: 46%;
+  margin-left: 4%;
+}
+.home .walk .walk-img img {
+  width: 100%;
+  height: auto;
+  border-radius: 10px;
+}
```
Comment (secantsquared) [inline css/index.css:370]: Appropriate and clear named classes.

File: less/index.less [hunk 8]
```diff
@@ -1,12 +1,12 @@
 // Import your less files here:
 
-@import: "variables.less";
-@import: "mixins.less";
-@import: "reset.less";
-@import: "global.less";
-@import: "navigation.less";
-@import: "footer.less";
-@import: "home-page.less";
+@import "variables.less";
```
Comment (secantsquared) [inline less/index.less]:
```suggestion 
@import "variables"
```
you can drop the less as it assumes it is less by default

File: less/mixins.less [hunk 9]
```diff
@@ -1 +1,8 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
```
Comment (secantsquared) [inline less/mixins.less:1]: Intelligent use of mixin

File: less/navigation.less [hunk 10]
```diff
@@ -1 +1,23 @@
...
+    flex-wrap: wrap; 
+    justify-content: space-around;
+
+        li {
+            font-size: 1.5em;
+            margin: 10px 25px;
+
+            a {
+                text-decoration: none;
+                color: @river-bend;
```
Comment (secantsquared) [inline less/navigation.less:25]: nice use of less var here

File: css/index.css (hunk 6 above)
Comment (fncischen) [reply css/index.css:223]: great rec! i'll do that. seems more efficient!

**Pull request merged**